client/components/tinymce/plugins/mentions/rest-client/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Module dependencies.
+ */
+import EventEmitter from 'events/';
+import wpcom from 'lib/wp';
+
+const debug = require( 'debug' )( 'calypso:tinymce:mentions' );
+
+export default class Client extends EventEmitter {
```

Comment:
What do you think about placing the data in the redux state tree and using actions & selectors?